### PR TITLE
Remove webpack

### PR DESCRIPTION
**Pull request by AlexB52**
I'm trying to remove Webpack and splits my assets back to something a bit more simple and easy to understand.

Here are the assets deployed on production:
CleanShot 2022-04-25 at 16 11 08@2x

Here are the assets bundled and minified from `sass` and `esbuild`
CleanShot 2022-04-25 at 16 11 19@2x

building the assets adds a hash but the size of the files is the same.

What's wrong?

### CODE REVIEW

**Comment by ndorfin** [comment]
Hey @AlexB52 - what are you using esbuild for?

**Comment by AlexB52** [comment]
@ndorfin I'm using `esbuild` to bundle my js files into a site.js. Is that wrong?

File: assets/javascripts/application.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ import { Application } from "stimulus"
 import { definitionsFromContext } from "stimulus/webpack-helpers"
```
Comment (ndorfin) [inline assets/javascripts/application.js:5]: What about use ES6 Modules here, and then forgoing any build step?

Comment (ndorfin) [reply assets/javascripts/application.js:5]: Does Stimulus come with a pre-minified version you could import once, and be done with it?

File: assets/stylesheets/primer.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+@import "../../node_modules/@primer/css/header/index.scss";
+@import "../../node_modules/@primer/css/labels/index.scss";
+@import "../../node_modules/@primer/css/loaders/index.scss";
+@import "../../node_modules/@primer/css/markdown/index.scss";
+@import "../../node_modules/@primer/css/popover/index.scss";
+@import "../../node_modules/@primer/css/progress/index.scss";
+@import "../../node_modules/@primer/css/select-menu/index.scss";
+@import "../../node_modules/@primer/css/subhead/index.scss";
+@import "../../node_modules/@primer/css/timeline/index.scss";
+@import "../../node_modules/@primer/css/toasts/index.scss"
```
Comment (ndorfin) [inline assets/stylesheets/primer.scss:46]: Do you use all of these?

**Comment by AlexB52** [comment]
@ndorfin Hmm that's weird. The deployed version on Netlify seems to have the same size. How come?
https://deploy-preview-37--alexbarret.netlify.app/
CleanShot 2022-04-25 at 16 17 50@2x

File: package.json [hunk 3]
```diff
@@ -1,26 +1,21 @@
 {
-  "name": "webpack-middleman",
+  "name": "portfolio-middleman",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (ndorfin) [inline package.json:7]: You could delete this line :P

**Comment by ndorfin** [comment]
Those look like brotli'd or gzipped responses, whereas your local dev server is not compressing your output.

File: assets/stylesheets/primer.scss (hunk 2 above)
Comment (AlexB52) [reply assets/stylesheets/primer.scss:46]: I used to have this:
```scss
@import '@primer/css';
```
I'm trying to remove Webpack for now independently from what I'm really using that said you're right. I could remove some. I know I'm using Box, Header and Markdown :p

Comment (ndorfin) [reply assets/stylesheets/primer.scss:46]: Ah, yeah, good to stay focused.

File: assets/javascripts/application.js (hunk 1 above)
Comment (AlexB52) [reply assets/javascripts/application.js:5]: I think I can remove this completely. It's a bit annoying to import each controller manually but good enough I guess

Comment (AlexB52) [reply assets/javascripts/application.js:5]: Not sure. What if Stimulus is not loaded before I get to register the controllers?

**Pull request merged**